App.js

```diff
@@ -5,19 +5,18 @@ import { createStackNavigator, StackActions, NavigationActions } from 'react-nav
 import HomeScreen from './screens/HomeScreen';
 import InputScreen from './screens/InputScreen';
 import ResultsScreen from './screens/ResultsScreen';
-import InputManual from './components/InputManual';
 
 
-export default createStackNavigator({
-  HomeScreen: {
-    screen: HomeScreen,
-  },
-  InputScreen: {
-    screen: InputScreen,
+
+
```

Comment:
Extra lines

Reply:
Yeap need to clean up! I tried to look for packages on atom to: 

- Remove extra lines
- Remove extra import not used in file

Definitely to add on Trellololo.